hikari/api/rest.py

```diff
@@ -5509,10 +5509,13 @@ async def fetch_ban(
         """
 
     @abc.abstractmethod
-    async def fetch_bans(
+    def fetch_bans(
         self,
         guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
-    ) -> typing.Sequence[guilds.GuildBan]:
+        *,
+        before: undefined.UndefinedOr[snowflakes.SnowflakeishOr[users.PartialUser]] = undefined.UNDEFINED,
+        after: undefined.UndefinedOr[snowflakes.SnowflakeishOr[users.PartialUser]] = undefined.UNDEFINED,
```

Comment:
This signature would prob be better like fetch_scheduled_event_users where its `newest_first: bool = False, start_at: undefined.UndefinedOr[snowflakes.SnowflakeishOr[users.PartialUser]]` which would make the note about before vs after unnecessary